template/6-non_functional_requirements.md

```diff
@@ -3,14 +3,20 @@
 ## Security, privacy, and data retention policies
 
 *Which are the applicable laws and regulations?*
+- Regulation from [Google Map API geolocation](https://developers.google.com/maps/documentation/geolocation/policies?hl=fr)
```

Comment:
I think that we should use the english version of the page

```suggestion
- Regulation from [Google Map API geolocation](https://developers.google.com/maps/documentation/geolocation/policies?hl=en)
```

Reply:
Good idea, thanks!